basis/common-router/src/main/java/com/softgroup/common/router/api/RequestHandler.java

```diff
@@ -1,7 +1,10 @@
 package com.softgroup.common.router.api;
 
+import com.softgroup.common.protocol.Request;
+import com.softgroup.common.protocol.Response;
 import org.springframework.stereotype.Component;
 
 @Component
 public interface RequestHandler extends Handler {

```

Comment:
Datamapper is needed only in abstract class, for converting, implementations just return Response object and spring takes care of converting. But yet I can't make spring deal with empty Data object cause it fails on empty beans. So is it better to return String instead of Response object?